SCH-2804 New loading effect setup

Youssefsh opened the pull request.

tenub reviewed (changes requested):
Looks great for the most part. Left a few minor comments.

I think that we should also resolve all of the `useEffect` missing dependency warnings before merging.

Youssefsh commented on components/src/components/layout/card/Card.tsx:
```diff
@@ -1,5 +1,9 @@
+import { useEmbed } from "../../../hooks";
+import { Flex, Loader } from "../../ui";
+import { useTheme } from "styled-components";
+import { TEXT_BASE_SIZE } from "../../../const";
```
@tenub what are we ordering based on? i updated it but i think we let the linter in the app do that automatically, maybe we should do that here too - but it's resolved

Youssefsh commented:
> Looks great for the most part. Left a few minor comments.
> 
> I think that we should also resolve all of the `useEffect` missing dependency warnings before merging.

@tenub Not sure about the `useEffect` dependencies here, since we did not add any of it in these changes, additionally:

1. We have to return the `<Column />` otherwise we can't dnd anything in the 3rd column in the builder, updated that accordingly so it does not return a `<Card />` wrapper around the `children`
2. Added a new color function: `hexToRGBA`
3. I'm now only exporting `<Success />` until we decide whether we need it or not ( what do you think? ), i think we might be able to re-use soon

tenub commented on components/src/components/layout/column/Column.tsx:
```diff
@@ -8,7 +8,11 @@ export interface ColumnProps extends React.HTMLProps<HTMLDivElement> {
 export const Column = forwardRef<HTMLDivElement | null, ColumnProps>(
   ({ children, basis, ...props }, ref) => {
-    return (
+    return Children.count(children) === 0 ? (
```
What about this?
```tsx
return (
  <StyledColumn ref={ref} {...props}>
    {Children.count(children) > 0 ? <Card>{children}</Card> : children}
  </StyledColumn>
);
```

Youssefsh replied:
@tenub yup that's a much better one liner, updated accordingly 🙏
